mmv1/products/datastream/api.yaml

```diff
@@ -351,6 +351,31 @@ objects:
         name: 'displayName'
         required: true
         description: Display name.
+      - !ruby/object:Api::Type::Enum
+        name: 'state'
+        description: |
+          State of the PrivateConnection.
+        output: true
+        values:
```

Comment:
In general we don't include this list for output-only fields (since it's not used for anything in that case) but it's fine to leave in.